pithos/plugins/notify.py

```diff
@@ -50,25 +48,20 @@ def send_notification(self, window, *ignore):
         notification = Gio.Notification.new(song.artist)
         notification.set_body(song.title)
         if song.artUrl:
-            notification.set_icon(Gio.FileIcon.new(Gio.File.new_for_uri(song.artUrl)))
+            icon = Gio.FileIcon.new(Gio.File.new_for_uri(song.artUrl))
+        else:
+            icon = Gio.ThemedIcon.new_with_default_fallbacks('audio-x-generic-symbolic')
```

Comment:
The backend doesn't support fallbacks, it hardcodes the first value: https://github.com/GNOME/glib/blob/5a73eef7498bacce9b5b176a927af8ad20d2947e/gio/gfdonotificationbackend.c#L302-L304

Reply:
Ok then I'll just use ```Gio.ThemedIcon.new('audio-x-generic')```

GNOME Shell auto-upgrades to symbolic icons.

Reply:
> GNOME Shell auto-upgrades to symbolic icons.

Oh nice.